package.json

```diff
@@ -28,17 +29,23 @@
     "postversion": "git push --follow-tags",
     "tdd": "yarn run test:mocha --watch",
     "test": "yarn run todos && yarn run lint && yarn run typecheck && yarn run test:mocha",
-    "test:mocha": "yarn run test:mocha:glob 'test/**/*test.js'",
-    "test:mocha:glob": "NODE_ENV=test mocha --opts mocha.opts",
+    "test:mocha": "yarn run test:mocha:glob 'test/**/*test.{js,jsx,ts,tsx}'",
+    "test:mocha:glob": "NODE_ENV=test mocha --opts mocha.opts --require @babel/polyfill --require .register.ts --extension ts,js,jsx,tsx",
```

Comment:
consider `test:mocha` → `test:unit` while you're here, then it will be fully compliant